README.md

```diff
@@ -105,7 +105,13 @@ would be represented as:
 The projection authority is some domain governing body that defines  projections. 
 The contents of this field are permissive and any string can be provided. In practice, it is preferable 
 to use a known authority. 
-Examples include `epsg`, `iau_2015`, and `ogc`.
+
+| Authority Name | proj:authority | URL |
+| -------------- | -------------- | --- |
+| COSMO          | `cosmo`          |     |
```

Comment:
Do we know anything about COSMO?

Reply:
I do not. I did a quick google and it looks like this is related to some polar data. Specifically, https://external.ogc.org/twiki_public/CRSdefinitionResolver/RotatedGridCrsDefinition. I'm not sure though.

Reply:
I think we should stick to whatever is presented by OGC.

Reply:
The COSMO information is super light. I wish there was more (or that I knew where to find it). Another reference here: https://external.ogc.org/twiki_public/CRSdefinitionResolver/CRSdiscussion. It appears to be for some sort of rotated meteorological data (which sounds quite in scope for STAC).

A good question then is, 'where is COSMO even coming from'?. I found it here, in a short list provided by OGC in their API. Drilling into that authority, it appears to be a single projection string for a global GCS for atmospheric data: http://www.opengis.net/def/crs/COSMO/0/100

So, maybe the thing to do is to include the link to that OGC page. I fear this is a really niche community using this projection and omitting might mean that that group fails to possibly benefit from STAC.

Reply:
> I fear this is a really niche community using this projection and omitting might mean that that group fails to possibly benefit from STAC.

They can simply use WKT2/PROJJSON, right? All the shortcodes that we have here are for convenience of the data provider and search (but are potentially an inconvenience for the [web] data clients). So in theory we could remove proj:epsg and the fields from this PR altogether and would not really loose any capabilities. You should be able to express all shortcodes also in WKT2/PROJJSON.

Reply:
Correct, `proj:epsg` should fully deprecate. The authority/code are important to encode the same info if someone isn't using the wkt2/projjson strings. It may (?) also be helpful if/when clients use something like [proj4js](http://proj4js.org).